zabbix/manifest.json

```diff
@@ -0,0 +1,33 @@
+{
+  "display_name": "zabbix",
+  "maintainer": "KosukeKamiya@users.noreply.github.com",
+  "manifest_version": "1.0.0",
+  "name": "zabbix",
+  "metric_prefix": "zabbix.",
+  "metric_to_check": "",
+  "creates_events": false,
+  "short_description": "",
```

Comment:
Let's include a brief description of this integration

Reply:
Added!